src/Symfony/Component/Ldap/Entry.php

```diff
@@ -22,9 +22,22 @@ class Entry
     public function __construct(string $dn, array $attributes = array())
     {
         $this->dn = $dn;
+        $this->resetValueIndex($attributes);
         $this->attributes = $attributes;
     }
 
+    /**
+     * Resets the index for array value
+     *
+     * @param array $attributes
+     */
+    public function resetValueIndex($attributes = array())
```

Comment:
this must be private. We don't want to expose it as a public API

Reply:
Yeah done.